src/main/kotlin/com/chattriggers/ctjs/minecraft/objects/KeyBind.kt

```diff
@@ -22,24 +21,27 @@ class KeyBind {
      */
     @JvmOverloads
     constructor(description: String, keyCode: Int, category: String = "ChatTriggers") {
-        for (key in Client.getMinecraft().gameSettings.keyBindings) {
-            if (key.keyCategory == category && key.keyDescription == description) {
-                Client.getMinecraft().gameSettings.keyBindings = ArrayUtils.removeElement(
-                    Client.getMinecraft().gameSettings.keyBindings, key
-                )
-                break
-            }
+        val possibleDuplicate = Client.getMinecraft().gameSettings.keyBindings.find {
+            I18n.format(it.keyDescription) == I18n.format(description) &&
+            I18n.format(it.keyCategory) == I18n.format(category)
```

Comment:
Is there a reason to format these? If the translation key is the same, the translation would also be the same, right?

Reply:
I just tested it, and these are the results of not comparing the description and categories formatted.

- Attack/Destroy, Gameplay => game crashes from error
- Attack/Destroy, key.categories.gameplay => creates a "duplicate" keybind (appears to have same name and category, but is completely separate)
- key.attack, key.categories.gameplay => game crashes from error
- key.attack, Gameplay => game crashes from error

Reply:
lit